src/header/css/header-menus.pcss

```diff
@@ -235,7 +235,370 @@
 
 /* Global Header Menus: Submenu
 /* ========================================================================== */
-	
+
+.esri-header-menus-flyout {
+	background: var(--mono-94);
+
+	&::after {
+		content: '';
+		display: table;
+		clear: both;
+	}
+
+	@media (--width-l) {
+		overflow: hidden;
+	}
+}
+
+.esri-header-menus-flyout--categories,
```

Comment:
Are you trying to do BEM here? Usually sub-components inside a block are `__` while modifiers are `--`.

Reply:
I am not trying to do BEM.. This is just my preference.. I don't like to use _ unless it's a two_word or something..